src/main/java/com/metsci/laproc/tools/PointInfoPanel.java

```diff
@@ -29,30 +29,37 @@ public PointInfoPanel() {
     }
 
     /**
-     * updates the rows of the point info panel
+     * updates the rows of the point info panel for each displayed graph
      */
     public void update(GraphPoint[] points){
-        Map<String, Double> data = points[0].getAnalytics();
+        this.panel.remove(pane);
+        pane = new JScrollPane();
```

Comment:
I want to explore making this not wonky in terms of how I do it but every time I have tried to pursue this option the panels refuse to redraw. There is probably some swing rule I need to follow to do it correctly but we should probably just explore that later.